files/build_templates/sonic_debian_extension.j2

```diff
@@ -40,6 +40,10 @@ FILESYSTEM_ROOT_USR_SHARE_SONIC_TEMPLATES="$FILESYSTEM_ROOT_USR_SHARE_SONIC/temp
 FILESYSTEM_ROOT_ETC="$FILESYSTEM_ROOT/etc"
 FILESYSTEM_ROOT_ETC_SONIC="$FILESYSTEM_ROOT_ETC/sonic"
 
+# Create the directoy. This is needed for Stretch and might not need
+# be needed for Buster. It should not harm anyways
+sudo mkdir -p $FILESYSTEM_ROOT_USR_LIB_SYSTEMD_SYSTEM
+
```

Comment:
Suggest to move this just below where all other system directories are created (lines 79-83)

Reply:
@jleveque Done